app/src/main/res/values/arrays.xml

```diff
@@ -21,4 +21,18 @@
         <item>1</item>
         <item>2</item>
     </string-array>
+    <string-array name="www_substitution_entries">
+        <item>Do not change subdomain part</item>
+        <item>Try with www. subdomain first, then without</item>
```

Comment:
FYI moolticute will always strip the www. , so we may want to remove this option  and the following 3 ones.